utoipa/src/openapi.rs

```diff
@@ -795,61 +780,10 @@ mod tests {
             .build();
 
         api_1.merge(api_2);
-        let value = serde_json::to_value(&api_1).unwrap();
-
-        assert_eq!(
-            value,
-            json!(
-                {
-                  "openapi": "3.1.0",
-                  "info": {
-                    "title": "Api",
-                    "version": "v1"
-                  },
-                  "paths": {
-                    "/ap/v2/user": {
-                      "get": {
-                        "responses": {
-                          "200": {
-                            "description": "Get user success 2"
-                          }
-                        }
-                      }
-                    },
-                    "/api/v1/user": {
-                      "get": {
-                        "responses": {
-                          "200": {
-                            "description": "Get user success"
-                          }
-                        }
-                      }
-                    },
-                    "/api/v2/user": {
-                      "post": {
-                        "responses": {
-                          "200": {
-                            "description": "Get user success"
-                          }
-                        }
-                      }
-                    }
-                  },
-                  "components": {
-                    "schemas": {
-                      "User2": {
-                        "type": "object",
-                        "properties": {
-                          "name": {
-                            "type": "string"
-                          }
-                        }
-                      }
-                    }
-                  }
-                }
-            )
-        )
+
+        assert_json_snapshot!(api_1, {
+            ".paths" => insta::sorted_redaction()
+        });
```

Comment:
Does this keep the `paths` sorted always in same order?

Reply:
yep. the `merge()` call caused them to be a bit random for some reason, but this fixed it.

Reply:
https://insta.rs/docs/redactions/ has more info on this :)